packages/bot-web-ui/src/assets/sass/transactions.scss

```
@@ -71,7 +71,7 @@
         padding: 8px 0;
 
         &-line {
-            height: 4px;
+            height: 1px;
```

The line should be 4px according to design, please revert.

```suggestion
            height: 4px;
```